http/digest/src/main/java/org/wildfly/security/http/digest/NonceManager.java

```diff
@@ -271,6 +273,13 @@ boolean useNonce(final String nonce, byte[] salt, int nonceCount) throws Authent
         }
     }
 
+    /**
+     * must be called in order to shut down executor
+     */
+    public void shutDown() {
```

Comment:
This is a very minor comment, but noticed that "shutDown" is used here but "shutdown" is used elsewhere. Would be good to be consistent.

Reply:
Fixed